docs/source/intervention_models/crm_mgmt/scenarios.rst

```diff
@@ -0,0 +1,64 @@
+[Thom-2013]_ 
+
+**Implementation:**
+
+Adherence to blood pressure lowering and statin medications over 1 year after initiation should be increased according to the above study, though in this scenario we will actually deliver the following combination: atorvastatin (10 mg), amlodipine (2.5 mg), losartan (25 mg), and hydrochlorothiazide (12.5 mg). Effect can persist beyond the length of the study.  
+[Munoz-NEJM]_ 
+
+**Scenarios:**
+	- Polypill 1.0: All individuals with IHD, past ischemic stroke, or prevalent PAD (who are not already on medications) receive atorvastatin (10 mg), amlodipine (2.5 mg), losartan (25 mg), and hydrochlorothiazide (12.5 mg) regardless of SBP or LDL-c levels. Medications lead to expected reduction in SBP and LDL-c. Adherence is increased 13% following receipt of those medications.  
+	- Polypill 0.5: Above scenario with, but a reduction to 50% of individuals meeting criteria receiving prescription for atorvastatin (10 mg), amlodipine (2.5 mg), losartan (25 mg), and hydrochlorothiazide (12.5 mg) regardless of SBP or LDL-c levels). Medications lead to expected reduction in SBP and LDL-c. Adherence among individuals receiving therapy is increased 13% following receipt of those medications. 
+
+
+4.1.3 Lifestyle modification education
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Under this scenario, lifestyle modification education regarding physical activity, healthy eating, weight loss, and tobacco cessation is utilized.  
+
+**Source information:**
+
+Individuals were enrolled in the diabetes prevention program if their BMI was >= 25, they had no known DM2, and their FPG was 100-125mg/dl (or HgbA1C 5.7-6.4). They attended a median of 14 community-based sessions over a median of 134 days. Sessions included lifestyle and behavior counselling with a focus on increasing moderate physical activity, healthy eating, and weight loss. Each additional session attended led to 0.31% loss of body weight.  
+[Ely-2017]_  
+
+Variations on the above intervention increased support time or added meal replacements for the first month or full year  
+[Metz-et-al-2000]_ 
+
+**Scenarios:**
+	- Lifestyle 1.0: Enrollment will occur following a routine health facility or primary care visit. Individuals with BMI >= 25 or FPG 100-125 mg/dl at the time of the visit will receive weekly sessions for 6 months followed by monthly sessions for 6 months. Each session attended will result in 0.3% loss of body weight (initially represented as BMI, but eventually to use weight when available). Adherence will decline in a linear fashion, with 50% of individuals continuing sessions at 6 months and all completing sessions by 1 year.  
+	- Lifestyle 0.5: Scenario described above but with 50% of adherence at initiation; adherence will decline in a linear fashion from that point. 
+
+
+4.1.4 Combined scenarios
+~~~~~~~~~~~~~~~~~~~~~~~~
+	- Combination 1.0: Combination of outreach 1.0, polypill 1.0, and lifestyle 1.0  
```

Comment:
specify when these levels of coverage should be attained, and if there should be an immediate scale up or a gradual scale up from baseline (and if gradual, describe how, e.g. linear scale up from baseline levels on 2022-01-01 to 100% coverage on 2023-01-01)

Reply:
have added to the outreach section; will add to the other two sections when I flesh them out more.